BinaryContract.sol

```diff
@@ -1,3 +1,62 @@
 pragma solidity >=0.4.22 <0.7.0;
 
-//TODO
\ No newline at end of file
+//TODO
+contract BinaryContract{
+
+    struct Transaction {
+        address from;
+        address to;
+        uint amount;
+        uint date;
+    } // this is only needed if we're keeping a transaction log inside a bin contract as well
+
+    struct ContractDebt{
+        address debtor;
+        uint amountOwned;
+    }
+
+
+    address playerOne;
+    address playerTwo;    
+    ContractDebt currentDebt;
+
+    Transaction[] binContractTransactionsLog; 
+    /* not sure if needed because the transactions are already saved in the 
+    group contract and a two person group i.e binary contract is just a two person group contract with a log already being made.
+    My question is: do we need to differentiate between a binary contract and a two person group? */ 
```

Comment:
BinaryContract is like GroupContract, except that:

    BinaryGroup has 2 members
    BinaryGroup has different attributes:
        - Debtor
        - Creditor

I think that we should have a Transaction log in the BinaryGroup, so that when a GroupContract is expired,
the GroupContract's TransactionsLog will look in both members' groups 
(let's say that we keep both GroupContracts and BinaryContracts in members' ProfileContract's "groups" attribute)
if such BinaryContract does not exist (where the members are both the Debtor and Creditor),
then it creates one.
else, it appends another Transaction to the BinaryGroup Transaction log, and updated its expiration date.